paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultManageScreenInteractorTest.kt

```diff
@@ -144,6 +145,44 @@ class DefaultManageScreenInteractorTest {
         }
     }
 
+    @Test
+    fun removeSecondToLastPaymentMethod_cantRemoveLastPm_cbcEligible_navsBackWhenEditingFinishes() {
```

Comment:
Should we add some tests for other scenarios as well?

Reply:
Sure! Added tests for:
- `allowsRemovalOfLastSavedPM` is true + the card is cbc eligible, we do not nav back when "done" is clicked
- when you remove the third from last card (so there are two remaining saved cards), we do not nav back when "done" is clicked

I think we have pretty good coverage of other edge cases